stablehlo/transforms/VhloLegalizeToStablehlo.cpp

```diff
@@ -174,20 +167,19 @@ class VhloToStablehloOpConverter : public OpConversionPattern<VhloOpTy> {
     // vhlo.case that uses a variadic number of regions which means an
     // additional argument for the generic builder.
     VhloToStablehloOp<VhloOpTy> stablehloOp;
-    if constexpr (std::is_same<VhloOpTy, vhlo::CaseOpV1>::value) {
+    if constexpr (std::is_same<VhloOpTy, vhlo::CaseOpV1>::value)
```

Comment:
if with else shouldn't drop braces

Reply:
The LLVM guide suggests dropping if the if/else has a single statement.

Reply:
Looked in LLVM codebase, and the rule applies to `if constexpr` as well.